solr/core/src/test/org/apache/solr/cli/TestExportTool.java

```diff
@@ -218,6 +231,66 @@ public void testVeryLargeCluster() throws Exception {
     }
   }
 
+  @Test
+  public void testWithBasicAuth() throws Exception {
+    String COLLECTION_NAME = "secureCollection";
+    String USER = "solr";
+    String PASS = "SolrRocksAgain";
+    final String SECURITY_JSON =
+        Utils.toJSONString(
+            Map.of(
+                "authorization",
+                Map.of(
+                    "class",
+                    RuleBasedAuthorizationPlugin.class.getName(),
+                    "user-role",
+                    singletonMap(USER, "admin"),
+                    "permissions",
+                    singletonList(Map.of("name", "all", "role", "admin"))),
+                "authentication",
+                Map.of(
+                    "class",
+                    BasicAuthPlugin.class.getName(),
+                    "blockUnknown",
+                    true,
+                    "credentials",
+                    singletonMap(USER, getSaltedHashedValue(PASS)))));
+
+    configureCluster(2)
+        .addConfig("conf", configset("cloud-minimal"))
+        .withSecurityJson(SECURITY_JSON)
+        .configure();
+
+    CloudSolrClient cloudSolrClient = cluster.getSolrClient();
+
+    CollectionAdminRequest.createCollection(COLLECTION_NAME, "conf", 8, 1)
```

Comment:
8 shards?!
To tame our tests, tests should have a comment justifying >1 nodes or >1 shards.  If there is a reason; justify it.

Reply:
total copy paste ;-)

Reply:
Oh, wait...  I found it..  The test is `testVeryLargeCluster` and it's a nightly...   so I think the is why....

Reply:
do you want another specific comment on the test....

Reply:
I made the change to reduce the number...